website/docs/foundations/elevation/partials/code/how-to-use.md

```diff
@@ -1,47 +1,52 @@
 ## How to use these styles
 
-There are two different ways to apply an "elevation" or "surface" effect to a UI element, via **design tokens** or via **CSS helper classes**.
+You can apply an `elevation` or `surface` effect to an element via **design tokens** or **CSS helper classes**.
 
-Which one to use will depend on the context: since both refer to the same design token values, from the design system perspective there is no difference between the two methods (it’s more of a preference for the "consumer" codebase).
+!!! Info
+
+Note that `border-radius` is not included with this token and needs to be set according to the specs of the UI element.
+!!!
 
 ### Design tokens
 
-You can use the "elevation" and "surface" [design tokens](./tokens) directly in your CSS definitions:
+Use the `elevation` and `surface` [design tokens](./tokens) directly in your CSS definitions. Note that it can only be used with the `box-shadow` property.
```

Comment:
Should this be 'Note that **they** should only be use...' since you reference design tokens plural?